UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/Bodypart/ReagentCirculatedComponent.cs

```diff
@@ -8,20 +10,21 @@ public class ReagentCirculatedComponent : BodyPartComponentBase<ReagentPoolSyste
 		[Tooltip(" doesn't contribute to the volume of the blood pool, Just used for How much bleeding happens That type of thing")]
 		public float Throughput;
 
+		public BloodType bloodType => bloodReagent as BloodType;
```

Comment:
Rename property 'bloodType' to match pascal case naming rules, consider using 'BloodType'.